src/array/random.jl

```diff
@@ -0,0 +1,30 @@
+using Random
```

Comment:
Nitpicking a bit, could we do explicit imports instead of `using`? It makes a bit easier to see where symbols come from.

Reply:
Sure, and I'll pull this one out to the top-level `Dagger.jl` file